src/extension.ts

```diff
@@ -162,7 +158,10 @@ async function init(context: vscode.ExtensionContext, git: GitApiImpl, credentia
 }
 
 export async function activate(context: vscode.ExtensionContext): Promise<GitApiImpl> {
+	extensionId = 'github.vscode-pull-request-github';
```

Comment:
there is actually `EXTENSION_ID` used below which should capture this and is changed in the build, it could just be marked as not const to be reused here

Reply:
Great, thanks!